main.js

```diff
@@ -0,0 +1,73 @@
+let bookList = [
+  {
+    id: Date.now().toString(36) + Math.random().toString(36).substr(2, 3),
+    title: 'Black Clover',
+    author: 'Yuki Tabata',
+  },
+];
+
+if (localStorage.getItem('books')) {
+  bookList = JSON.parse(localStorage.getItem('books'));
+}
+
+if (localStorage.getItem('inputTitle')) {
+  document.querySelector('.title-input').value = localStorage.getItem('inputTitle');
+  document.querySelector('.author-input').value = localStorage.getItem('inputAuthor');
+}
+
+function showBooks() {
+  const showBooksSection = document.querySelector('.show-books-section');
+  let bookRow = '';
+  if (bookList.length === 0) {
+    showBooksSection.innerHTML = '<h1>Awesome books</h1>';
+  } else {
+    bookList.forEach((book) => {
+      bookRow += `
+                <p>${book.title}</p>
+                <p>${book.author}</p>
+                <button class="${book.id}">Remove</button>
+                <hr>
+            `;
+      showBooksSection.innerHTML = `
+                <h1>Awesome books</h1>
+                ${bookRow}
+            `;
+    });
+  }
+
+  const removeBookButton = document.querySelectorAll('.show-books-section button');
+  removeBookButton.forEach((button) => {
+    button.addEventListener('click', () => {
+      /* eslint-disable no-use-before-define */
```

Comment:
- Please remove all disabled linters, rememeber that linters help us ensure that we write code that is clean and easy to maintain. Please fix this 🙏